cluster/get-kube.sh

```diff
@@ -183,6 +193,9 @@ if [[ ${KUBE_VERSION} =~ ${KUBE_CI_VERSION_REGEX} ]]; then
   # Override KUBERNETES_RELEASE_URL to point to the CI bucket;
   # this will be used by get-kube-binaries.sh.
   KUBERNETES_RELEASE_URL="${KUBERNETES_CI_RELEASE_URL}"
+elif [[ ${KUBE_VERSION} =~ ${KUBE_BAZEL_VERSION_REGEX} ]]; then
+  KUBE_VERSION=${KUBE_VERSION#bazel/}
```

Comment:
Can we just add an `export I_KNOW_WHAT_I_AM_DOING=true` type option and expect `kubetest` to set `KUBE_VERSION` and `KUBERNETES_RELEASE_URL` correctly?

This seems more straight forward than all this convoluted parsing